src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -694,7 +732,7 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
         case _ => Nil
       }
       val dependencies = upstream.flatten ++ downstream
-      val dependenciesResolved = dependencies.isEmpty || dependencies.forall(isDone)
+      val dependenciesResolved = dependencies.isEmpty || dependencies.forall(d => isDone(d) || isPreempted(d))
```

Comment:
This doesn't look right to me - if a call gets preempted then any calls which depend on it can now start?